OpenRA.Game/Map/MapCache.cs

```diff
@@ -85,36 +100,44 @@ public void LoadMaps()
 				}
 
 				mapLocations.Add(package, classification);
+				mapDirectoryTrackers.Add(new MapDirectoryTracker(mapGrid, package, classification));
 			}
 
-			var mapGrid = modData.Manifest.Get<MapGrid>();
 			foreach (var kv in MapLocations)
 			{
 				foreach (var map in kv.Key.Contents)
 				{
-					IReadOnlyPackage mapPackage = null;
-					try
-					{
-						using (new Support.PerfTimer(map))
-						{
-							mapPackage = kv.Key.OpenPackage(map, modData.ModFiles);
-							if (mapPackage == null)
-								continue;
+					LoadMap(map, kv.Key, kv.Value, mapGrid);
+				}
+			}
```

Comment:
Similarly the braces around the `LoadMap` call can go since it's on one line.

Reply:
ok

Reply:
Can also remove the next level of outer braces, so:
```c#
foreach (...)
    foreach (...)
        LoadMap(...);
```